captum/attr/_core/layer/layer_deep_lift.py

```diff
@@ -308,8 +308,8 @@ def __init__(self, model, layer):
                           Currently, it is assumed that both inputs and ouputs of
                           the layer can only be a single tensor.
         """
-        super(DeepLiftShap, self).__init__(model)
-        super(LayerDeepLift, self).__init__(model, layer)
+        DeepLiftShap.__init__(self, model)
```

Comment:
nit: here the order of calls looks different from previous ones. 
`
LayerDeepLift.__init__(self, model, layer)
DeepLiftShap.__init__(self, model)
`

Reply:
Will switch, thanks!